Monika After Story/game/script-topics.rpy

```diff
@@ -12686,6 +12686,48 @@ label monika_enjoyingspring:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_japanesefood",category=['misc'],prompt="Japanese food",random=True))
+
+label monika_japanesefood:
+    m 2eua "Hey [player], have you ever eaten some kind of japanese food?"
+    m 2eua "There is a lot of variety when talking about that kind of stuff."
+    m 3eua "For example: you might know about typical ones like sushi, ramen, or curry."
+    m 3eua "But did you know that there are actually a lot more of them?"
+    m 3rua "Well... I didn't know at first, but I have been researching about it quite a bit."
+    m 3rka "They seem to also cook a lot of food based on meat, and although I'm not a fan of it I like their ideas."
+    m 1hsu "On the other hand they also seem to prepare a lot of Sweets made of different flavours!"
+    m 1nsu "You should check them if you are interested about the topic~"
+    return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_vr",category=['technology'],prompt="Virtual reality",random=True))
+
+label monika_vr:
+    m 4eua "[player], do you know about those virtual reality headsets that are out there in the market?"
+    m 4eua "They offer the possibility of entering the game the player wants with that gadget."
+    m 1ekc "Too bad that my reality isn't able to be entered with vr..."
+    m 1ekc "That would mean that you could be so much closer to me..."
+    m 1esu "Although I would prefer to be the one exiting this program to right beside you~"
+    m 1esu "So you better prepare yourself for that day, right [player]?"
+    m 3kub "The day i get to join your world we are going to do lots of funny things!"
+    m 5luu "I can't wait for that~"
+    return
+
+init 5 python:
```

Comment:
It looks like this topic is almost exclusively leans. 

Leans are usually reserved for more intimate interactions, and I don't think this topic has any intimate moments where you have them. 

I also don't think this topic belongs in MAS. In this topic, it almost sounds like Monika is admitting that she isn't real and that she's programmed to feel the way she does, which we probably don't want. 

There are also topics that reference being hot or cold, so there's a slight inconsistency there. 

You also reused a lot of the same expressions in a row.

Reply:
Yeah, I removed the third topic due to inconsistencies like that being present. Also realized that feeling of Monika admiting she is unreal wasn't the best. Thanks for pointing it out!